facebook_connector.py

```diff
@@ -68,8 +70,15 @@ def get_cleaned_fully_merged_messages(strip_html_content=True,
     me = chats.user
     addresses = set()
     messages = []
+    # Suppressing warning on BS4 that shows up when trying to parse non HTML content
+    # Some messages might contain HTML tags others might not and we don't want to display
+    # these warnings to the user
     warnings.filterwarnings("ignore", category=UserWarning, module='bs4')
```

Comment:
Oy, you're right.  Just clarify in the comment that the real issue occurs when the text messae solely contains a URL